### PR TITLE
WS2 1688 - configure cropping image styles to crop aspect ratio by default

### PR DESCRIPTION
**Pull request by Bob-E**
### Description

**Issue:** It’s not obvious to user that crop bounding box must be clicked on in order to apply cropping, and can be saved regardless.

**Solution:** Reconfigure image styles that include the cropping tool so that they “Scale and crop” to a specific aspect ratio if the cropper is not clicked on before saving.

modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_lge.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_med.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_sml.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_1_1_lge.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_1_1_med.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_1_1_sml.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_lge.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_med.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.block_image_sml.yml
modified:   web/modules/webspark/webspark_blocks/config/install/image.style.video_poster_image.yml
modified:   web/modules/webspark/webspark_blocks/webspark_blocks.install

- Recommend link back to original Media item from component configuration modal window so user can update cropping if the default cropping was applied and not to their liking (I'll create a new ticket)

- Note for service team: If client’s Drupal site uses a subtheme of renovation, in other words, a sub-sub-theme, then /image/image--crop-thumbnail.html.twig must be copied from renovation to the sub-sub-theme folder in order for the blue crop lines to appear on the cropping tool.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1688)

### Checklist

- [X] Design updates match [Web Standards](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/) and [Unity Design System](https://unity.web.asu.edu)
- [X] Solution is documented on the Jira ticket
- [X] QA steps to verify have been included on the Jira ticket
- [X] No new PHP or JS errors
- [X] No accessibility issues are introduced with this update

### Verified in browsers 

- [X] Chrome
- [X] Safari
- [X] Firefox
- [x] Edge

### CODE REVIEW

**Comment by mlsamuelson** [comment]
[Visit Site](https://pr-144-webspark-ci.pantheonsite.io/)

Created multidev environment [pr-144](https://dashboard.pantheon.io/sites/aec2f75f-35eb-41f8-abe7-95b292415259#pr-144) for webspark-ci.

File: web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_lge.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ dependencies:
     - crop.type.16_9
   module:
     - crop
+_core:
```
Comment (juanmitriatti) [inline web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_lge.yml:8]: I think this entire key `_core`  added by the CMS must be removed from the PR.
Right @mlsamuelson , @bebaps, @tbutterf ?

Comment (Bob-E) [reply web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_lge.yml]: @juanmitriatti I've gone ahead and removed the `_core` and hash lines from the configs.

**Review by bebaps — APPROVED** [review]
Tested locally, and confirmed the crop before and after this update.

**Pull request merged**